### PR TITLE
Implement functionality

### PR DESCRIPTION
**Pull request by karlenDimla**
Task/Issue URL: https://app.asana.com/0/0/1205651876758230/f

### Description
See attached task description

### Steps to test this PR
https://app.asana.com/0/0/1205768861130372/f

### CODE REVIEW

**Comment by karlenDimla** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3650** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3650" target="_blank">Graphite</a> 
    * **PR #3673** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3673" target="_blank">Graphite</a> 
      * **PR #3714** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3714" target="_blank">Graphite</a> 
        * **PR #3719** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3719" target="_blank">Graphite</a> 
      * **PR #3674** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3674" target="_blank">Graphite</a> 
        * **PR #3723** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3723" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3723?utm_source=stack-comment).

**Comment by karlenDimla** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3650** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3650" target="_blank">Graphite</a> 
    * **PR #3673** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3673" target="_blank">Graphite</a> 
      * **PR #3714** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3714" target="_blank">Graphite</a> 
        * **PR #3719** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3719" target="_blank">Graphite</a> 
      * **PR #3674** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3674" target="_blank">Graphite</a> 
        * **PR #3723** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3723" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3723?utm_source=stack-comment).

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt [hunk 1]
```diff
@@ -39,7 +39,7 @@ interface WgVpnControllerService {
     ): List<EligibleServerInfo>
 }
 
-const val NETP_ENVIRONMENT_URL = "https://controller.netp.duckduckgo.com"
+const val NETP_ENVIRONMENT_URL = "https://staging1.netp.duckduckgo.com"
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt:42]: Will revert once changes are deployed to prod

Comment (karlenDimla) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt]: Done

**Review by aitorvs — APPROVED** [review]
LGTM

**Pull request merged**